page-home.php

```diff
@@ -0,0 +1,194 @@
+      $tomorrow = $now->addDays(1);
+      $midnight = $tomorrow->startOf('day');
+      $midnight_timestamp = strtotime($midnight->format());
+
+      $forthcoming_events = new WP_Query(array(
+        'post_type' => 'event',
+        'posts_per_page' => 3,
+
+        'meta_query' => array(
+          array(
+            'key'     => '_igv_event_datetime',
+            'value'   => $midnight_timestamp,
+            'type'    => 'numeric',
+            'compare' => '>',
+          ),
+        ),
+      ));
+
+      if ($forthcoming_events->have_posts()) {
+    ?>
+
+    <div class="grid-row margin-top-basic margin-bottom-small">
+      <?php render_divider('Forthcoming Live Events'); ?>
+    </div>
+
+    <div class="grid-row">
+    <?php
+        while ($forthcoming_events->have_posts()) {
+          $forthcoming_events->the_post();
+          get_template_part('partials/custom-pages/home/event-forthcoming');
+        }
+      }
+    ?>
+    </div>
+
+    <div class="grid-row margin-top-basic margin-bottom-large">
+      <?php render_divider('<a href="' . home_url('live') . '" class="link-button">More Live Events +</a>'); ?>
+    </div>
+
+    <?php
+      $ad1_text = IGV_get_option('_igv_home_options', '_igv_ad1_text');
+      $ad1_image_id = IGV_get_option('_igv_home_options', '_igv_ad1_image_id');
+      $ad1_link = IGV_get_option('_igv_home_options', '_igv_ad1_link');
+      $ad1_link_external = IGV_get_option('_igv_home_options', '_igv_ad1_link_external');
+
+      if ($ad1_text && $ad1_image_id) {
+    ?>
+    <div class="grid-row margin-bottom-mid">
+      <?php render_ad($ad1_text, $ad1_image_id, $ad1_link, $ad1_link_external); ?>
+    </div>
+    <?php
+      }
+    ?>
+
+    <div class="grid-row margin-bottom-small">
+      <?php render_divider('Sound & Vision'); ?>
+    </div>
+
+    <div class="grid-row justify-center margin-bottom-basic">
+    <?php
+      $overrides = IGV_get_option('_igv_home_options', '_igv_override_events');
+      $posts = 3;
+      $not_in = array();
+
+      if ($overrides) {
+        $overrides = explode(', ', $overrides);
+
+        $posts = $posts - count($overrides);
+        $not_in = $overrides;
+      }
+
+      $past_args = array(
+        'post_type' => 'event',
+        'posts_per_page' => $posts,
+        'post__not_in' => $not_in,
+
+        'meta_query' => array(
+          array(
+            'key'     => '_igv_event_datetime',
+            'value'   => $midnight_timestamp,
+            'type'    => 'numeric',
+            'compare' => '<',
+          ),
+        ),
+      );
+
+      $past_events = get_posts($past_args);
+
+      $filter_ids = function($post) {
+        return $post->ID;
+      };
+
+      $past_event_ids = array_map($filter_ids, $past_events);
+
+      if ($overrides) {
+        $past_events = array_merge($overrides, $past_event_ids);
+      } else {
+        $past_events = $past_event_ids;
+      }
+
+      global $post;
+      foreach ($past_events as $post_id) {
```

Comment:
Some of the IDs are manually selected. So it would have to merge two WP_Queries after they are done, that seemed more or less the same as doing this. But perhaps a new WP_Query just with that array of IDs is possible? (also this code is based on already merged page-live.php)